public/app/features/alerting/unified/components/notification-policies/ContactPointSelector.tsx

```diff
@@ -76,8 +79,7 @@ export const ContactPointSelector = ({
         <IconButton
           name="sync"
           onClick={onClickRefresh}
-          aria-label="Refresh contact points"
```

Comment:
I think you removed this line by mistake?

Reply:
The button still gets a label from the tooltip value when its a string :)

![image](https://github.com/user-attachments/assets/c6fd5643-1d10-40f2-ad7c-8de615746862)
https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/IconButton/IconButton.tsx#L64